src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```
@@ -562,9 +562,11 @@
     </ItemGroup>
   </Target>
 
-  <!-- Native AOT properties should be shown by default, except for WPF, WinForms, and class library projects. -->
+  <!-- Native AOT properties should be shown by default for projects targeting .NET 8 or higher, except for WPF and WinForms projects -->
   <PropertyGroup>
-    <ShowNativeAOTProperties Condition="'$(ShowNativeAOTProperties)' == '' and '$(UseWPF)' == 'false' and '$(UseWindowsForms)' == 'false' and $(OutputType) != 'Library'">true</ShowNativeAOTProperties>
+    <ShowNativeAOTProperties Condition="'$(ShowNativeAOTProperties)' == ''
+                             and ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_TargetFrameworkVersionWithoutV)' >= '8.0')">true</ShowNativeAOTProperties>
```

nit: the parentheses suggest some kind of precedence that's not actually there. Consider removing them.

```suggestion
    <ShowNativeAOTProperties Condition="'$(ShowNativeAOTProperties)' == ''
                             and '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_TargetFrameworkVersionWithoutV)' >= '8.0'">true</ShowNativeAOTProperties>
```